lib/sidekiq_alive/config.rb

```diff
@@ -24,6 +25,7 @@ def set_defaults
       @registered_instance_key = 'SIDEKIQ_REGISTERED_INSTANCE'
       @queue_prefix = :sidekiq_alive
       @server = ENV['SIDEKIQ_ALIVE_SERVER'] || 'webrick'
+      @custom_liveness_probe = proc {}
```

Comment:
```suggestion
      @custom_liveness_probe = proc { true }
```
The default should return `true` if not will fail to write the liveness probe unless is overridden.